VAMobile/src/screens/HomeScreen/ProfileScreen/PersonalInformationScreen/PersonalInformationScreen.tsx

```diff
@@ -89,10 +89,10 @@ const PersonalInformationScreen: FC<PersonalInformationScreenProps> = ({ navigat
     }
   }
 
-  if (screenError || getDemographicsError || getGenderIdentityOptionsError || getPersonalInfoError) {
+  if (personalInformationInDowntime || getDemographicsError || getGenderIdentityOptionsError) {
```

Comment:
I removed `getPersonalInfoError` as a condition for showing the error component because this screen is mainly for giving user's access to edit demographics data. The only way information from the `v2/user` endpoint is used is in displaying the DOB, and when that information isn't available, a message under the DOB tab is displayed saying this information is unavailable. 

Plus, chances are by the time a user navigates to this screen, the user's personal information has already been fetched from the home screen, so even if the personal info query were to fail here, we'd still have the previously fetched personal info stored in cache, and that would be used for the DOB